src/containers/Webmap/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FC, useEffect, useState} from 'react';
+import axios from 'axios';
+import {Icon} from 'leaflet';
+import {Map, Marker, Popup, TileLayer} from 'react-leaflet';
+import './Webmap.scss';
+
+export const bankIcon = new Icon({
+  iconSize: [20, 20],
+  iconUrl: '/bank1.svg',
+});
+
+export const validatorIcon = new Icon({
+  iconSize: [20, 20],
+  iconUrl: '/validator.svg',
+});
+
+const Webmap: FC = () => {
+  const bankEndpoint = 'http://157.230.75.212/banks?limit=30&offset=0';
```

Comment:
I think these endpoints can be extracted as constants under the `src/constants`?

Reply:
Do you mean like making a new file in src/constants called "api-endpoints.ts" for example and doing this?
```
export const BANK_ENDPOINT = 'http://157.230.75.212/banks?limit=30&offset=0';
export const iPINFO_ENDPOINT = 'http://ip-api.com/batch/';
export const VALIDATOR_ENDPOINT = 'http://157.230.75.212/validators?limit=30&offset=0';
```

And then accessing them in index.tsx like 

```
import * as apiEndpoints from '../../constants/api-endpoints';
const resultBatch = await axios.post(apiEndpoints.iPINFO_ENDPOINT, validatorIps);
```

Reply:
yep!

Reply:
cool I implemented this